ext/TrixiConvexECOSExt.jl

```diff
@@ -160,6 +220,98 @@ function Trixi.bisect_stability_polynomial(consistency_order, num_eig_vals,
         gamma_opt = [gamma_opt]
     end
 
+    undo_normalization!(gamma_opt, num_stage_evals,
+                        consistency_order, consistency_order)
+
+    return gamma_opt, dt
+end
+
+# Specialized routine for PERK4.
+# For details, see Section 4 in 
+# - D. Doehring, L. Christmann, M. Schlottke-Lakemper, G. J. Gassner and M. Torrilhon (2024).
+# Fourth-Order Paired-Explicit Runge-Kutta Methods
+# [DOI:10.48550/arXiv.2408.05470](https://doi.org/10.48550/arXiv.2408.05470)
+function Trixi.bisect_stability_polynomial_PERK4(num_eig_vals,
+                                                 num_stage_evals,
+                                                 dtmax, dteps, eig_vals, cS3;
+                                                 verbose = false)
+    dtmin = 0.0
+    dt = -1.0
+    abs_p = -1.0
```

Comment:
All of this is restricted to `Float64`.

- Would it make sense to generalize it to also allow `Float32`?
- Shall we mention the restriction somewhere in the docs?